src/jobs/index.html

```diff
@@ -61,201 +61,55 @@
+    
+    <p>Lastly, while we like to move fast, we have processes to make everyone's life easier and use tools to make remote life better. This is a start-up, but you always get paid on time and you aren't living in a lonely bubble. We use Slack for communication, Tandem for hanging out, Clubhouse for project management, Front for support, Confluence for internal documentation, and Github for code and public documentation.</p>
+    
+    <h5>Our Products</h5>
+    <ul>
     <li>Web Application: <a target="_blank" rel="noopener" href="https://mycrypto.com/">https://mycrypto.com/</a></li>
-    <li>Desktop Applications (Electron): <a target="_blank" rel="noopener" href="https://download.mycrypto.com/">https://download.mycrypto.com/</a></li>
-    <li>Kickass Developer/Ecosystem Tools: <a target="_blank" rel="noopener" href="https://overflow.mycrypto.com/">https://overflow.mycrypto.com/</a></li>
-    <li>Mobile applications....soon™</li>
-  </ul>
-  <p>Our codebase was recently completely rewritten to use React/Redux/Typescript which means you won’t be maintaining legacy code here. 😃 Everything we do is open-source and licensed under MIT. Check it out: <a target="_blank" rel="noopener" href="https://github.com/mycryptohq/">https://github.com/mycryptohq/</a>.</p>
-  <h5>Things You Probably Care About</h5>
-  <ul>
+    <li>Desktop Application (Electron): <a target="_blank" rel="noopener" href="https://download.mycrypto.com/">https://download.mycrypto.com/</a></li>
+    <li>Ambo (iOS now, Android soon): <a target="_blank" rel="noopener" href="https://ambo.io/">ambo.io</a></li>
+    <li>Developer/Ecosystem Tools: <a target="_blank" rel="noopener" href="https://overflow.mycrypto.com/">overflow.mycrypto.com</a></li>
+    <li>CryptoScamDB: <a target="_blank" rel="noopener" href="https://cryptoscamdb.org/">cryptoscamdb.org</a></li>
+    <li>FindETH: <a target="_blank" rel="noopener" href="https://findeth.io/">https://findeth.io/</a></li>
+    </ul>
+    <p>Our code is fresh and we primarily use React & Typescript, meaning you won’t be maintaining legacy code. 😃 Everything we do is open-source and licensed under MIT. Check it out: <a target="_blank" rel="noopener" href="https://github.com/mycryptohq/">https://github.com/mycryptohq/</a>.</p>
+    
+    <h5>Things You Probably Care About</h5>
+    <ul>
     <li>We’re a decentralized (remote) team 🚀 with core operations based in Los Angeles.</li>
-    <li>This is a fast-paced, sink-or-swim environment.</li>
+    <li>We have team members in the US, Canada, UK, Netherlands, France, and Slovenia.</li>
+    <li>Weekly check-in calls are between 9am-11am Pacific Time. Everything else is async.</li>
+    <li>We move fast and encourage autonomy. This means you are responsible for managing your time and priorities. It also means you get to work on things you want to work on, when you want to work on them.</li>
```

Comment:
" It also means you get to work on things you want to work on, when you want to work on them."

Remove this part. For the most part its true but it's not legally binding that they always get to "work on things they want to work on" and "when they want to work" - their contract will take precedence on their timing/hours and priorities/tasks.